apps/web/frontend/components/timeSheetView/workHoursSheet.tsx

```diff
@@ -4,6 +4,7 @@ import useInfiniteScroll from 'react-infinite-scroll-hook'
 
 import { Spinner } from '@progwise/timebook-ui'
 
+import { PageHeading } from '../pageHeading'
```

Comment:
I guess this import is no longer needed? 🙂

Reply:
You are right. I removed it.